usagov/usagov-2021
# USAGOV-1575-mariadb-change-breaks-db-load: Accomodate dump file changes in the latest MariaDB version

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1575

## Description


1. Accomodate more info in header of SQL dump file, before the utf8mb4 line is seen
2. Remove the newly added, script-breaking, crazy-making line in the dump file, given to us by MariaDB. Thank you, Maria.
3. Allow db-dump-deploy (and therefore, also downsync-cms) to work w/ dump files that have not been scrubbed.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions


1. Be in your local dev environment's terminal, logged into Cloud.gov
2. Change to the Prod CF space:  `cf t -s prod`
3. Download a db dump known to contain the new (bad) MariaDB format:  `bin/snapshot-backups/db-snapshot-download prod USAGOV-1569.prod.9292.post-deploy`
4. Change to the Dev-DR CF space: `cf t -s dev-dr`
5. Deploy the tainted db dump to Dev-DR (hint: this PR should remove the offending dump line, allowing this command to successfuly complete):  `bin/snapshot-backups/db-dump-deploy dev-dr USAGOV-1569.prod.9292.post-deploy`

### Requires New Config

### Requires New Content

### Validation Steps
## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
